src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    String lineBreak = "---------------------------------------------";
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public String getTypeIcon() {
+        return "";
+    }
```

Comment:
Perhaps you can combine the square brackets into these methods so you don't have to repeat them for each of the different tasks, for example:
```
public String getStatusIcon() {
    return (isDone ? "[X]" : "[ ]");
}
```